low-level/framework/utils/utility.py

```diff
@@ -58,3 +60,55 @@ def is_env_vm(self):
             logger.warning("Error while reading whether env is vm or not, assuming VM env : {e}")
         return is_vm
 
+    @staticmethod
+    def send_command(command: str, fail_on_error=True):
+        # Note: This function uses subprocess to execute commands, scripts which are not possible to execute
+        # through any python routines available. So its usage MUST be limited and used only when no other
+        # alternative found.
+        output, error, returncode = SimpleProcess(command).run()
+        if returncode != 0:
+            print("command '%s' failed with error\n%s" % (command, error))
+            if fail_on_error:
+                sys.exit(1)
+            else:
+                return str(error)
+        return str(output)
+
+    @staticmethod
+    def call_script(script_dir: str, args: list):
+        script_args_lst = [script_dir]+args
+        is_error = subprocess.call(script_args_lst, shell=False)
+        if is_error:
+            sys.exit(1)
+
+    @staticmethod
+    def _initialize_dbus():
+        """Initialization of dbus object."""
+        system_bus = dbus.SystemBus()
+        systemd1 = system_bus.get_object('org.freedesktop.systemd1', '/org/freedesktop/systemd1')
+        dbus_manager = dbus.Interface(systemd1, 'org.freedesktop.systemd1.Manager')
+        return dbus_manager
+
+    @staticmethod
+    def enable_disable_service(service, action = 'enable'):
+        """Enable/Disable systemd services."""
+        dbus_manager = Utility._initialize_dbus()
+        if action == 'enable':
+            dbus_manager.EnableUnitFiles([f'{service}'], False, True)
+        else:
+            dbus_manager.DisableUnitFiles([f'{service}'], False)
+        dbus_manager.Reload()
+
+    @staticmethod
+    def systemctl_service_action(service, action = 'start'):
+        """Start/Stop/Restart systemctl services."""
+        dbus_manager = Utility._initialize_dbus()
+        if action == 'start':
+            dbus_manager.StartUnit(f'{service}', 'fail')
+        elif action == 'stop':
+            dbus_manager.StopUnit(f'{service}', 'fail')
+        elif action == 'restart':
+            dbus_manager.RestartUnit(f'{service}', 'fail')
+        else:
+            print(f"Invalid action: f'{service}' :Please provide an appropriate action name for the service.")
```

Comment:
I feel like this should be in a try block.

Reply:
seems this is open still

Reply:
@cdeshmukh Addressing this through https://github.com/Seagate/cortx-utils/pull/161 in cortx-utils

Reply:
ok